game-core/src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1436,7 +1375,7 @@ private Matches() {}
   }
 
   static Predicate<Unit> transportIsTransporting() {
-    return transport -> TransportTracker.isTransporting(transport);
+    return TransportTracker::isTransporting;
```

Comment:
Is there any reason to keep this as a method? I'm curoius why it's not just inlined.

Reply:
Inlined.